src/pages/practice/storytelling-framework.md

```diff
@@ -0,0 +1,167 @@
+---
+templateKey: practice-page
+title: Storytelling Framework
+subtitle: Framework to morph your idea into an impactful, emotive story that
```

Comment:
suggested alternative for the subtitle/description:
Morph ideas into impactful, emotive stories to resonate with your audience.